doc/stages/ogr.json

```diff
@@ -0,0 +1,13 @@
+
+{
```

Comment:
I think we should have 

```
"type":"ogr"
```

as part of the definition of this thing

Reply:
Optional or required?

Reply:
Required. It's not a free-form thing.

Anyone adding them to `readers.ept` or `readers.copc` without the `type` will get an error and have to update their pipelines.

Reply:
Wouldn't this break existing pipelines?

Reply:
Just mine 😄 

(I don't think the `ogr` option in `readers.ept` and `readers.copc` is widely used or known about)